src/db/prisma/migrations/migration_lock.toml

```diff
@@ -0,0 +1,3 @@
+# Please do not edit this file manually
+# It should be added in your version-control system (i.e. Git)
+provider = "postgresql"
```

Comment:
Newline at end please

Reply:
So fix it upstream :) https://github.com/prisma/prisma-engines/blob/45e6871398e3e66d16be7a12a029938abdc1709c/migration-engine/connectors/migration-connector/src/migrations_directory.rs#L53-L74

Reply:
100%